### PR TITLE
First draft of a news post describing Linda's work in the GUI

### PR DESCRIPTION
**Pull request by veroandreo**
This is a first draft of an upcoming series of 2 news items describing Linda's and Caitlin's work. Originally, we thought of showing off only the student grant results but it's moreover a path that led there, IMO. So, this is the first draft + screenshots provided by @lindakladivova with some minor edits by me. 

There is still room for edition and beautification with highlights, links, mentors names, and such. I'd also add a sentence of two about the mini-grants per se and a closure about the opportunity or so. Maybe it could even be a short intro in third person, describing the student grants and GRASS intention with it, and then Linda's words. As it is her path, I think it's ok that it is written in the first person. Still, I'd like to hear others' opinion about it. 

Help, comments, edits are more than welcome :)

### CODE REVIEW

**Comment by veroandreo** [comment]
Preview snippet:
image

File: content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+<a href="/images/gallery/gui/grass8_firststartup.png">
+  <img src="/images/gallery/gui/grass8_firststartup.png"
+   alt="GRASS GIS at first launch"
+   title="GRASS GIS at first launch"
+   width="70%" style="float:right;padding-left:20px;padding-top:20px">
+</a>
+
+I started contributing to GRASS GIS in the summer of 2020 as part 
+of the project *"Creation of a new GRASS GIS startup mechanism"*, 
+which I applied for as a complete programming illiterate. Although
```
Comment (petrasovaa) [inline content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md:20]:
```suggestion
I started contributing to GRASS GIS in the summer of 2020 as part 
of the Google Summer of Code (GSoC) project
*"Creation of a new GRASS GIS startup mechanism"*, 
which I applied for as a complete programming illiterate. Although
```

File: content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+I felt a bit clumsy at first, with the great help of my mentors I 
+managed to significantly improve the GRASS data catalog. 
+Looking backwards now, this was the starting point for all the
```
Comment (petrasovaa) [inline content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md:23]:
```suggestion
Looking back now, this was the starting point for all the
```

File: content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+upcoming fundamental changes in the graphical user interface (GUI).
+
+In the following months, as part of my master thesis, I focused on 
+the topic of a **new startup mechanism** and **the improvement 
+of the first-time user experience**. The whole effort led to the complete 
+elimination of the hard-to-grasp startup screen and the implementation
+of a new first-time user mode, which made GRASS significantly more
+accessible to new users after almost forty years.
```
Comment (petrasovaa) [inline content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md:31]:
```suggestion
of the first-time user experience**. The whole effort led to
eliminating the hard-to-grasp startup screen and implementing
a new first-time user mode, which made GRASS significantly more
accessible to new users.
```

File: content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+online community surveys I created as part of my master thesis. 
+Thanks to them, other thorny topics emerged among which I was mostly 
+interested in the need to unify GRASS into a **Single-Window layout**. 
+From the very beginning, I wanted to make working with GRASS as pleasant
+as possible, but until then I have largely focused on new users. 
+I realized that Single-Window would be a big implementation nut that 
+would completely transform the work in the software and break down the 
+established habits. Remembering a great experience from 2020, I eventually
+decided to create a mock-up graphic design, which was then the basis for a 
+successful application to the next Google Summer of Code (GSoC).
```
Comment (petrasovaa) [inline content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md:50]:
```suggestion
successful application to the next GSoC.
```

File: content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+What helped a lot in steering the further implementations were the 
+online community surveys I created as part of my master thesis. 
+Thanks to them, other thorny topics emerged among which I was mostly 
+interested in the need to unify GRASS into a **Single-Window layout**. 
+From the very beginning, I wanted to make working with GRASS as pleasant
+as possible, but until then I have largely focused on new users. 
+I realized that Single-Window would be a big implementation nut that 
+would completely transform the work in the software and break down the 
+established habits. Remembering a great experience from 2020, I eventually
```
Comment (petrasovaa) [inline content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md:48]:
```suggestion
I realized that implementing Single-Window would be rather difficult,
requiring significant refactoring of large part of the GUI code.
Recalling the great experience from GSoC 2020, I eventually
```

File: content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+successful application to the next Google Summer of Code (GSoC).
+
+<a href="/images/gallery/gui/grass8_single_window_startupGUI.jpg">
+  <img src="/images/gallery/gui/grass8_single_window_startupGUI.jpg"
+   alt="Single-Window GUI at first launch"
+   title="Single-Window GUI at first launch"
+   width="70%" style="float:right;padding-left:15px;padding-top:20px">
+</a>
+
+Based on the illustration, my mentors and me managed to rewrite part of the
```
Comment (petrasovaa) [inline content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md:59]:
```suggestion
Based on the illustration, my mentors and I managed to rewrite part of the
```

**Review by petrasovaa — APPROVED** [review]
I added couple suggestions.

**Comment by veroandreo** [comment]
> I added couple suggestions.

Suggestions applied, thanks @petrasovaa !! :smiley:

File: content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+   title="Mock-up for GSoC 2021 proposal"
+   width="60%" style="float:left;padding-right:20px;padding-top:20px">
+</a>
+
+What helped a lot in steering the further implementations were the 
+online community surveys I created as part of my master thesis. 
+Thanks to them, other thorny topics emerged among which I was mostly 
+interested in the need to unify GRASS into a **Single-Window layout**. 
+From the very beginning, I wanted to make working with GRASS as pleasant
+as possible, but until then I have largely focused on new users.
```
Comment (lindakladivova) [inline content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md:46]: Maybe "had largely focused?" I am not sure about the right grammar...

**Comment by lindakladivova** [comment]
I have read it again and added a couple of minor suggestions. Overall, I think it looks good.
Thank you girls @veroandreo @petrasovaa for the review :-)!
And also thank you Vero for compiling the webpage.

File: content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md (hunk 7 above)
Comment (petrasovaa) [reply content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md:46]: I agree

Comment (veroandreo) [reply content/news/2022_05_20_Result_student_grant_Linda_Kladivova.md]: done! thanks for catching it :)

**Pull request merged**